miio/airhumidifier.py

```diff
@@ -7,6 +7,10 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class AirHumidifierException(Exception):
```

Comment:
This should derive from `DeviceException` (looks like we have some other exceptions too, which do not do this currently) to allow easy catching of all exception from this library no matter which device is being used.